pkgs/tools/system/kanata/default.nix

```diff
@@ -28,7 +37,7 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/jtroo/kanata";
     license = licenses.lgpl3Only;
     maintainers = with maintainers; [ linj ];
```

Comment:
Would you like to add yourself to the maintainer list?  This makes it easier for you to maintain this package.  For example, our CI will [ping][1] you when it creates a PR to bump the version.


Here is the [doc][2] and an [example][3] of doing so.

[1]: https://github.com/NixOS/nixpkgs/pull/276116
[2]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[3]: https://github.com/NixOS/nixpkgs/commit/7f10f172110477ea263d63b6c793ebd8637eed63

Reply:
I am definitely up for helping. Is it best to do that in a separate PR or as part of this one?

Reply:
You can do that in this PR with two commits:

- in the first commit, you add your entry to `maintainer-list.nix`
- in the second commit, you change the current commit by adding yourself to `maintainers` list.